Source/santactl/Commands/SNTCommandRule.m

```diff
@@ -45,6 +46,49 @@ + (NSString *)shortHelpText {
   return @"Manually add/remove/check rules.";
 }
 
+- (BOOL)isWellformedSHA256:(NSString *)hash {
+  if (hash.length != 64) {
+    return NO;
+  }
+
+  NSCharacterSet *hexChars =
+    [NSCharacterSet characterSetWithCharactersInString:@"0123456789ABCDEF"];
+
+  return [@"" isEqualToString:[hash stringByTrimmingCharactersInSet:hexChars]];
+}
+
+- (BOOL)isWellformedTeamID:(NSString *)teamID {
+  if (teamID.length != 10) {
+    return NO;
+  }
+
+  NSCharacterSet *validChars = [NSCharacterSet alphanumericCharacterSet];
+
+  return [@"" isEqualToString:[teamID stringByTrimmingCharactersInSet:validChars]];
+}
+
+- (void) checkIfRuleIsWellformed:(SNTRule *)rule{
+  switch (rule.type) {
+    case SNTRuleTypeBinary:
+    case SNTRuleTypeCertificate:
+      if (![self isWellformedSHA256:[rule.identifier uppercaseString]]) {
```

Comment:
We should uppercase the identifier when parsing the arguments instead so it's just one location it needs to be handled.